src/common/Cargo.toml

```diff
@@ -15,6 +15,8 @@ auto_enums = "0.7"
 bitflags = "1.3.2"
 byteorder = "1"
 bytes = "1"
+sysinfo = "0.26"
+num_cpus = "0.2"
```

Comment:
Shall we also remove this unused dependency?

Reply:
Have been removed, will push the latest code.